### ISSUE TITLE
CNN doesn't working

### ISSUE THREAD

**Issue by inerplat** [inerplat/JoYuris-Classifier#3]
acc and loss aren't proceed in the desired direction😨
```
Epoch 1/50
15/15 [==============================] - 29s 2s/step - loss: 8.6590 - acc: 0.4667 - val_loss: 10.0290 - val_acc: 0.3778
Epoch 2/50
15/15 [==============================] - 22s 1s/step - loss: 10.0290 - acc: 0.3778 - val_loss: 8.5963 - val_acc: 0.4667
Epoch 3/50
15/15 [==============================] - 23s 2s/step - loss: 8.9545 - acc: 0.4444 - val_loss: 8.9545 - val_acc: 0.4444
Epoch 4/50
15/15 [==============================] - 23s 2s/step - loss: 9.3127 - acc: 0.4222 - val_loss: 9.3127 - val_acc: 0.4222
Epoch 5/50
15/15 [==============================] - 26s 2s/step - loss: 9.6709 - acc: 0.4000 - val_loss: 10.3872 - val_acc: 0.3556
Epoch 6/50
15/15 [==============================] - 33s 2s/step - loss: 9.3127 - acc: 0.4222 - val_loss: 11.2452 - val_acc: 0.3023
Epoch 7/50
15/15 [==============================] - 24s 2s/step - loss: 8.9545 - acc: 0.4444 - val_loss: 9.6709 - val_acc: 0.4000
Epoch 8/50
15/15 [==============================] - 23s 2s/step - loss: 10.7454 - acc: 0.3333 - val_loss: 9.6709 - val_acc: 0.4000
Epoch 9/50
15/15 [==============================] - 23s 2s/step - loss: 9.3127 - acc: 0.4222 - val_loss: 8.5963 - val_acc: 0.4667
Epoch 10/50
15/15 [==============================] - 22s 1s/step - loss: 7.8800 - acc: 0.5111 - val_loss: 8.5963 - val_acc: 0.4667
Epoch 11/50
15/15 [==============================] - 22s 1s/step - loss: 10.7454 - acc: 0.3333 - val_loss: 9.3710 - val_acc: 0.4186
Epoch 12/50
15/15 [==============================] - 25s 2s/step - loss: 10.4174 - acc: 0.3537 - val_loss: 9.3127 - val_acc: 0.4222
Epoch 13/50
15/15 [==============================] - 24s 2s/step - loss: 8.5963 - acc: 0.4667 - val_loss: 10.7454 - val_acc: 0.3333
Epoch 14/50
15/15 [==============================] - 23s 2s/step - loss: 7.5218 - acc: 0.5333 - val_loss: 8.9545 - val_acc: 0.4444
Epoch 15/50
15/15 [==============================] - 25s 2s/step - loss: 11.4618 - acc: 0.2889 - val_loss: 9.3127 - val_acc: 0.4222
Epoch 16/50
15/15 [==============================] - 23s 2s/step - loss: 8.9545 - acc: 0.4444 - val_loss: 7.1219 - val_acc: 0.5581
Epoch 17/50
15/15 [==============================] - 23s 2s/step - loss: 10.0290 - acc: 0.3778 - val_loss: 11.4618 - val_acc: 0.2889
Epoch 18/50
15/15 [==============================] - 23s 2s/step - loss: 9.3127 - acc: 0.4222 - val_loss: 9.6709 - val_acc: 0.4000
Epoch 19/50
15/15 [==============================] - 22s 1s/step - loss: 9.6709 - acc: 0.4000 - val_loss: 8.9545 - val_acc: 0.4444
Epoch 20/50
15/15 [==============================] - 23s 2s/step - loss: 11.4618 - acc: 0.2889 - val_loss: 9.3127 - val_acc: 0.4222
Epoch 21/50
15/15 [==============================] - 23s 2s/step - loss: 10.0290 - acc: 0.3778 - val_loss: 9.7458 - val_acc: 0.3953
Epoch 22/50
15/15 [==============================] - 22s 1s/step - loss: 8.2381 - acc: 0.4889 - val_loss: 10.3872 - val_acc: 0.3556
Epoch 23/50
15/15 [==============================] - 24s 2s/step - loss: 9.3127 - acc: 0.4222 - val_loss: 10.3872 - val_acc: 0.3556
Epoch 24/50
15/15 [==============================] - 24s 2s/step - loss: 9.2929 - acc: 0.4234 - val_loss: 8.2381 - val_acc: 0.4889
Epoch 25/50
15/15 [==============================] - 24s 2s/step - loss: 10.0290 - acc: 0.3778 - val_loss: 9.3127 - val_acc: 0.4222
```

**Comment by inerplat** [comment]
Probably, there are many problems, but the biggest problem is that the training data is bad condition.
Therefore, it is necessary to visually screen images.

![1](https://user-images.githubusercontent.com/21287813/62951337-c386c400-be24-11e9-979e-d06fd19e41de.png)

[1] Some images that are detected may not have a face
[2] If several people are detected, the desired person may not appear.

**Comment by inerplat** [comment]
I've visually screened the images and changed the batch size, then they work normally.🤗

```
Epoch 22/200 30/30 [==============================] - 44s 1s/step - loss: 0.3041 - acc: 0.8818 - val_loss: 0.4196 - val_acc: 0.8701
Epoch 00022: val_loss improved from 0.44526 to 0.41962, saving model to ./model.h5
```
